### ISSUE TITLE
[13pt] Create sample inputs ESIP directory to work against two HUCS

### ISSUE THREAD

**Issue by RobHanna-NOAA** [NOAA-OWP/inundation-mapping#936]

**Comment by RobHanna-NOAA** [comment]
Add Watchers @BradfordBates-NOAA

**Comment by CarsonPruitt-NOAA** [comment]
I created a directory on our on-prem machine called `sample_inputs` with the bare minimum (744MB) needed for a pipeline run on 12090301. There's a README explaining that the data is incomplete and users can only use `-u 12090301` as a HUC arg. Please take a look at the README file and do another test to make sure it runs successfully.

**Comment by CarsonPruitt-NOAA** [comment]
FYI, I was able to run using this directory by simply adding another volume mount to my docker container like so `-v /home/user/sample_inputs/:/data/inputs`. I don't know whether that's easier than renaming the directory to `/inputs`, but it's another option.

**Comment by RobHanna-NOAA** [comment]
oh ya.. that seems easier. I forgot about that one.

**Comment by RobHanna-NOAA** [comment]
sample_inputs uploaded to ESIP

**Comment by RobHanna-NOAA** [comment]
Another update: We have a small correction to make on one of the URLs.
is: aws s3 cp --recursive s3://noaa-nws-owp-fim/hand_fim/outputs/fim_4_3_11_0/12090301 \
    /your_local_folder_name/12090301 --request-payer requester

sb: aws s3 cp --recursive s3://noaa-nws-owp-fim/hand_fim/fim_4_3_11_0/12090301 \
    /your_local_folder_name/12090301 --request-payer requester   (remove the /outputs)

**Comment by RobHanna-NOAA** [comment]
Yes other update: Annie at ESIP changed the permissions to our bucket so absolutely no credentials are required anymore. However.. the syntax changes a little (notice the ending arguments).  `aws s3 ls s3://noaa-nws-owp-fim/hand_fim/ --no-sign-request`.  And make sure we have that new tag on the end of the copy tags as well.

**Comment by RobHanna-NOAA** [comment]
Can I now recommend that we tell our users that they get all inputs from us. I think we can still provide links to the original sources, we want to make sure they know that these are raw and not pre-processed and may not be directly compatible with running against fim code.

**Comment by RobHanna-NOAA** [comment]
I think this is done. I finished it a month or more ago. I will double check it.

**Comment by jcphill** [comment]
FYI, the sample_inputs directory does not work because it does not contain the included_huc8_withAlaska.lst file needed since https://github.com/NOAA-OWP/inundation-mapping/blame/82b5e422da3e59c988899994ac97fcfbcb4f4b63/src/check_huc_inputs.py#L18 and there is no error if the glob matches no files.

**Comment by RobHanna-NOAA** [comment]
Yes. the sample inputs directory does not work as we have not been able to update it for a while. I will bring it up to my tech lead and see what we can do. Might take a while though but I will keep you posted.

**Comment by mluck** [comment]
There is some code to generate sample data in the `dev-sample-data` branch, but it hasn't been modified to work for Alaska. See step 1 in https://github.com/NOAA-OWP/inundation-mapping/pull/1178.

**Comment by RobHanna-NOAA** [comment]
Can I recommend we make a new issue card and get it assigned to someone. @mluck : Do you think you have time to look into it? If not, I can teach someone else how to do it. Thoughts?
